sdk/eventgrid/Azure.Messaging.EventGrid/src/Generated/Models/AcsMessageReceivedEventData.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+
+namespace Azure.Messaging.EventGrid.SystemEvents
+{
+    /// <summary> Schema of the Data property of an EventGridEvent for a Microsoft.Communication.AdvancedMessageReceived event. </summary>
+    public partial class AcsMessageReceivedEventData : AcsMessageEventData
```

Comment:
Should this actually be named AcsMessagesReceivedEventData to align with the Azure.Communication.Messages library?
(as suggested by @billwert)

Reply:
I think `Message` reads better than `Messages` in most of the names here and the other ACS service prefixes in system events are all singular too.  I'd leave it as you have it here unless folks feel very strongly.

Reply:
My understanding is that this is not used in the REST API anywhere.